qutip/core/options.py

```diff
@@ -40,5 +40,5 @@ class CoreOptions:
         # general relative tolerance
         "rtol": 1e-12,
         # use auto tidyup absolute tolerance
-        "auto_tidyup_atol": 1e-12
+        "auto_tidyup_atol": 1e-14
```

Comment:
Why did the default tolerance change?

Reply:
Now that the cleaning step is in the operation itself, it will probably be called more often in loops so I expect the numerical error will accumulate faster. This is to balance that.

Reply:
Sounds good.